src/jesse.app.src

```diff
@@ -1,10 +1,14 @@
 %%-*- mode: erlang -*-
 
 { application, jesse, [ {description, "jesse"}
-                      , {vsn, git}
+                      , {vsn, "0.1.3"}
```

Comment:
No you are right, this change was simply needed because the git version of the push to hex (due to the changed commit) would have been `0.1.3+gitsomethingsomethignsomething`